documentation/notifications.md

```diff
@@ -0,0 +1,202 @@
+# Notifications
+
+Sometimes, it is necessary to communicate with our users outside of the application. In order to do that, we can send emails and SMS messages using [GC Notify](https://notification.canada.ca/). This is done on the server in the `api` Laravel project.
+
+> **Note**
+> It may be useful to read over the [GC Notify documentation](https://documentation.notification.canada.ca/en/)
```

Comment:
Allow docs site to choose language
```suggestion
> It may be useful to read over the [GC Notify documentation](https://documentation.notification.canada.ca/)
```